roles/etcd/tasks/system_container.yml

```diff
@@ -15,6 +15,57 @@
       {%- endif -%}
       {% endfor -%}
 
+- name: Check etcd system container package
+  command: >
+    atomic containers list --no-trunc -a -f container=etcd -f backend=ostree
+  register: etcd_result
+
+- name: Unmask etcd service
+  systemd:
+    name: etcd
+    state: stopped
+    enabled: yes
+    masked: no
+    daemon_reload: yes
+  register: task_result
+  failed_when: task_result|failed and 'could not' not in task_result.msg|lower
```

Comment:
I copied this chunk from other rules dealing with `systemd` services:

```bash
$ git grep "failed_when: task_result|failed and 'could not' not in task_result.msg|lower"
roles/etcd/tasks/main.yml:    failed_when: task_result|failed and 'could not' not in task_result.msg|lower
roles/openshift_master/tasks/main.yml:  failed_when: task_result|failed and 'could not' not in task_result.msg|lower
roles/os_firewall/tasks/firewall/firewalld.yml:  failed_when: task_result|failed and 'could not' not in task_result.msg|lower
roles/os_firewall/tasks/firewall/iptables.yml:  failed_when: task_result|failed and 'could not' not in task_result.msg|lower
```

Is it fine to leave it as it is?